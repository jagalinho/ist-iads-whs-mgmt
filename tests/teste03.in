a 60fdba63 24
a c614e44d 2
a ff4095a1 38
a c614e44d 36
a 5d5c3b04 28
a 376f7f4d 43
a 376f7f4d -10
m
a 5d5c3b04 10
m
r 5d5c3b04
m
r c614e44d
m
r ff4095a1
m
x
