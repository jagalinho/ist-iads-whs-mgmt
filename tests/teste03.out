c614e44d 38
5d5c3b04 38
c614e44d 38
ff4095a1 38
376f7f4d 33
2
